config.yaml.defaults

```diff
@@ -440,3 +440,21 @@ spectrum_types:
    - host
    - host_center
   default: source
+
+# define the additional passbands that are not in SNCosmo
+# the names must be unique. The wavelength array should
```

Comment:
Add link to existing bandpasses? https://sncosmo.readthedocs.io/en/stable/bandpass-list.html